app/styles/style.css

```diff
@@ -131,4 +137,270 @@ body {
     text-decoration: none;
     display: flex;
   }
+
+  .jobs {
+    padding: 24px;
+  }
+
+  .card {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    flex-direction: column;
+    height: auto;
+    width: 100%;
+    background-color: #fff;
+    border-radius: 16px;
+    border: 1px #dfe1e6 solid;
+    margin-top: 114px;
+  }
+
+  .card-container {
+    padding: 16px;
+  }
+
+  .card .card-img {
+    width: 100%;
+  }
+
+  .card-title-text {
+    margin: 0;
+    margin-top: 16px;
+  }
+
+  .job-img {
+    border-radius: 8px;
+    width: 100%;
+  }
+
+  .card-title {
+    font-family: PoppinsBold;
+    font-size: 24px;
+    line-height: 44px;
+    font-weight: 700;
+  }
+
+  .card-subtitle {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    justify-content: space-between;
+    align-items: center;
+    width: 236px;
+    font-family: Poppins;
+  }
+
+  .el1 {
+    color: #344563;
+    font-size: 13px;
+    line-height: 16px;
+    font-weight: 600;
+  }
+
+  .el2 {
+    color: #7a869a;
+    font-size: 13px;
+    line-height: 16px;
+    font-weight: 600;
+  }
+
+  .dividor {
+    background-color: #c1c7d0;
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+  }
+
+  .card-body {
+    width: 100%;
+    margin: 0;
+  }
+
+  .card-body p {
+    margin-top: 4px;
+  }
+
+  .card-description {
+    font-family: Poppins;
+    color: #344563;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .stack-section {
+    display: flex;
+  }
+
+  .stack-container {
+    padding-right: 8px;
+  }
+
+  .tech-stack {
+    color: #6070ff;
+    background-color: #ebebff;
+    font-family: Poppins;
+    height: 24px;
+    padding: 6px 12px 6px 12px;
+    font-size: 12px;
+    align-content: center;
+    border-radius: 8px;
+  }
+
+  .card-footer {
+    margin-top: 48px;
+    padding-bottom: 16px;
+  }
+
+  .show-button {
+    padding: 12px;
+    text-decoration: none;
+    border: 1px #6070ff solid;
+    border-radius: 8px;
+    color: #6070ff;
+    font-family: Poppins;
+  }
```

Comment:
To enhance user experience, I recommend adding the options for Active, Hover, and Pressed states to your design. These states can help users better understand the functionality of your interface and provide visual feedback when interacting with it. By incorporating these states, you can improve usability and make your design more intuitive. 😄